pkg/lorabasics/doc.txt

```diff
@@ -0,0 +1,15 @@
+/**
+ * @defgroup pkg_lorabasicsmodem   LoRa Basics
+ * @ingroup  pkg
+ * @brief    LoRa Basics Modem
+ *
+ * # Introduction
+ *
+ * Put an introduction of the package here
```

Comment:
Could you add some general information to the pkg? Also it would be good to mention that you think it could be done without the use of the `ral` here, as a possible TODO cleanup for someone else (or you) to do later.